TTHAnalysis/python/tools/nanoAOD/susySOS_modules.py

```diff
@@ -205,23 +239,31 @@ def clean_and_FO_selection_SOS_noBtag(lep,year):
                                          or (abs(lep.pdgId)==13 and lep.softId ) )
 
 def fullCleaningLeptonSel(lep,year):
```

Comment:
The `fullCleaningLeptonSel*` definitions should be updated to the new FO definition.

Reply:
The are now obsolete, since we switched to one tight definition everywhere.